### PR TITLE
Getting Started edits

### PR DESCRIPTION
**Pull request by kevinchristianson**
#### What? Why?

Our group ran into different issues setting up the code base and running the unit tests. After completing this process we think that there are a few small changes that could be made to the getting started guide. Our changes include alternate commands as well as more information for getting tests to pass like checking that the application.yml file's timezone matches the computer's.

#### What should we test?
This shouldn't require any additional testing

#### Release notes
Updated GETTING_STARTED.md to include more specific setup instructions and help troubleshoot potential issues.

Changelog Category: Changed

#### Documentation updates
This is a documentation update which shouldn't require any additional changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
-    "phantomjs-prebuilt": "~2.1.7",
-    "karma": "~0.13.22",
-    "karma-jasmine": "~0.3.8",
     "jasmine-core": "~2.4.1",
+    "karma": "^3.0.0",
+    "karma-coffee-preprocessor": "~0.3.0",
+    "karma-jasmine": "~0.3.8",
     "karma-phantomjs-launcher": "~1.0.0",
-    "karma-coffee-preprocessor": "~0.3.0"
+    "phantomjs-prebuilt": "~2.1.7"
```
Comment (sauloperez) [inline package.json]: This must be exactly what was described in https://github.com/openfoodfoundation/openfoodnetwork/pull/3002#issuecomment-437234713. You can check @Matt-Yorkley 's answer to know how to get rid of them in this branch. Don't hesitate to reach us out on the #dev channel if you need help.

Comment (haseleyi) [reply package.json]: Done!

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -93,7 +97,23 @@ You can run rubocop against your changes using:
...
+```
+$ createuser --superuser --pwprompt ofn
+Enter password for new role: f00d
+Enter it again: f00d
+$ createdb open_food_network_dev --owner=ofn
+$ createdb open_food_network_test --owner=ofn
+```
+If these commands succeed, you should be able to [continue the setup process](#get-it-running).
+
+### Resources
```
[the remark this answers is not in the document]
Comment (sauloperez) [reply GETTING_STARTED.md]: :point_up: still needs to be fixed

[the remark this answers is not in the document]
Comment (haseleyi) [reply GETTING_STARTED.md]: Sorry, thanks for catching that!

File: GETTING_STARTED.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ Then the tests can be run with:
     
 Note: If your OS is not explicitly supported in the setup guides then not all tests may pass. However, you may still be able to develop. Get in touch with the [#dev][slack-dev] channel on Slack to troubleshoot issues and determine if they will preclude you from contributing to OFN.
 
-Additionally, make sure that the time zone on your machine matches the one defined in `config/application.yml`. If there is a discrepancy, it may cause some RSpec tests to fail.
+Additionally, make sure that the time zone on your machine matches the one defined in `config/application.yml` (it should be Melbourne). Setting your machine's time zone to any other zone has the potential to cause test failures.
```
Comment (mkllnk) [inline GETTING_STARTED.md]: I think the part about test failures got solved in https://github.com/openfoodfoundation/openfoodnetwork/pull/2948. Can you verify that and take it out if solved?

Comment (haseleyi) [reply GETTING_STARTED.md]: Done!

**Comment by mkllnk** [comment]
Thank you!

**Pull request merged**